app/views/saasbook/_topbar.html.erb

```diff
@@ -0,0 +1,88 @@
+                    <a class="dropdown-button" href="#" download>
+                        <button type="button"
+                            class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
+                            data-placement="left">.html
+                        </button>
+                    </a>
+                    <!-- Download PDF via print -->
+                    <a class="dropdown-button">
+                        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
+                            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf
+                        </button>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Source interaction buttons -->
+            <a class="annotation-button" ><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" title="Make annotation"><i
+                        class="fa fa-comment"></i></button></a>
+
+            <a class="highlighter-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" title="Highlighter"><i
+                        class="fas fa-highlighter"></i></button></a>
+            <!-- Full screen (wrap in <a> to have style consistency -->
+
+            <a class="full-screen-button">
+                <button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
+                    title="Fullscreen mode">
+                    <i class="fas fa-expand"></i>
+                </button>
+            </a>
+
+
+
+            <!-- Launch buttons -->
+
+        </div>
+        <!-- user_github_omniauth_authorize_path -->
+        <div class="d-none d-md-block col-md-2 py-1 show">
+            <% if user_signed_in? %>
+                <a class="user-page-button" href= "user-page-path%>">
+                    <button type="submit" class="btn btn-default" title="User page" 
+                            data-toggle="tooltip" data-placement="bottom">
+                        <%= current_user.full_name %>
+                        <%= image_tag("http://github.com/#{current_user.full_name}.png", width: "15%")%>
+                    </button>
+                </a>
+            <% else %>
+                <a class="login-button" href= "<%= user_github_omniauth_authorize_path %>">
+                    <button type="submit" class="btn btn-default" title="Sign in with Github" 
+                            data-toggle="tooltip" data-placement="bottom">
+                        Sign in with GitHub
+                    </button>
+                </a>
+            <% end %>
+        </div>
+
+        
+            
+        
+
+
+    </div>
+</div>
```

Comment:
new lkine here too, you can just click the triple dot and edit in github since its a small change